### ISSUE TITLE
[slider] Vertical slider incorrectly exposed as horizontal in the accessibility tree in Chrome<124

### ISSUE THREAD

**Issue by oliviertassinari** [mui/base-ui#635]
### Steps to reproduce

Steps:
1. Open https://master--base-ui.netlify.app/components/react-slider/#vertical
or open https://mui.com/material-ui/react-slider/#vertical-sliders

### Current behavior

<img width="931" alt="SCR-20240918-mzoi" src="https://github.com/user-attachments/assets/2019d44f-511a-4526-bc02-613a9e93daa3">

### Expected behavior

<img width="930" alt="SCR-20240918-mzvb" src="https://github.com/user-attachments/assets/f71769e3-3c0d-48a5-a870-c80d9b281e92">

### Context

https://issues.chromium.org/issues/40739626 was fixed 3 years ago after Sebastian reported it, so we should now be able to remove the content in https://mui.com/material-ui/react-slider/#vertical-sliders and apply the style. This style is needed because aria-orientation is no correctly interpreted per https://issues.chromium.org/issues/40736841.

I noticed this in https://github.com/mui/base-ui/pull/636, it was weird.

Closing this issue also means getting https://mui.com/material-ui/react-slider/#vertical-sliders fixed. I opened this issue in Base UI to be at the root, for the Slider component owner.

### Your environment

<details>
  <summary><code>npx @mui/envinfo</code></summary>

```
  Don't forget to mention which browser you used.
  Output from `npx @mui/envinfo` goes here.
```
</details>

**Search keywords**: -

**Comment by mj12albert** [comment]
![Screenshot 2024-09-20 at 11 54 26 AM](https://github.com/user-attachments/assets/36e97e20-a3d1-4ef5-a5e2-02b0bf7bb37c)

As of 2023 Chromium plans to deprecate `webkit-appearance: slider-vertical` 😅 https://issues.chromium.org/issues/40261389

The recommendation is to use [`writing-mode`](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_writing_modes/Vertical_controls) instead, though for `<input type="range">` it doesn't match Base UI's browser support yet

For older browsers that we need to support, there are still (non-standard) properties to fall back to: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_writing_modes/Vertical_controls#creating_vertical_range_sliders_in_older_browsers

**Comment by oliviertassinari** [comment]
@mj12albert oh, I didn't know interesting. I guess it's https://github.com/mdn/content/pull/32142.

**Comment by mj12albert** [comment]
We discussed this and decided to:
- Set `writing-mode` in the built-in styles
- In the docs, recommend `webkit-appearance: slider-vertical` if older Chrome support is needed. We don't want to ship this with a console warning, and hopefully the usage share of Chrome<124 will decline fast.

**Comment by oliviertassinari** [comment]
Great first step.

I'm reopening per:

> Closing this issue also means getting https://mui.com/material-ui/react-slider/#vertical-sliders fixed. I opened this issue in Base UI to be at the root, for the Slider component owner.

There is nothing about Material Design in fixing this in Material UI, so I believe we want the two things to be done in coordination, by the same component owner.

**Comment by mj12albert** [comment]
The bug where arrow keys are reversed in Chromium for vertical sliders is fixed now: https://issues.chromium.org/issues/40739626#comment9

Here's a sandbox: https://codesandbox.io/p/sandbox/xhmkxy

Though supported browser versions are different between Base and Material UI right now, do you agree with generally doing the same:
- add CSS `writing-mode` to the built-in styles
- document `-webkit-appearance: slider-vertical` for older browsers
- remove the content related to the reversed arrow keys

@oliviertassinari @aarongarciah

**Comment by aarongarciah** [comment]
@mj12albert not sure I follow which styles are planned to ship as part of Base UI vs Material UI.

@oliviertassinari if this issue is solved in Base UI, let's close it and open one a new one in Material UI. It doesn't make sense to keep an issue open because a project that's supposed to adopt Base UI hasn't adopted it yet. I think this is a signal that thinking about "component owners" is not a good model.

**Comment by colmtuite** [comment]
> It doesn't make sense to keep an issue open because a project that's supposed to adopt Base UI hasn't adopted it yet.

Of course, we should stop this practice of leaving issues lingering around for ages. Once it's resolved for Base UI, it should be closed.

> I think this is a signal that thinking about "component owners" is not a good model.

We don't subscribe to the idea of "component owners" in the Core team. The way we operate is the Base UI team is responsible for shipping an accessible, flexible component, and the MUI team is responsible for wrapping it.

@mj12albert If MUI Slider wraps Base UI Slider, then won't the built-in styles be the same automatically? Also, MUI Slider styles will be bundled, so we don't need to document styling?

**Comment by mj12albert** [comment]
> let's close it and open one a new one in Material UI

I've opened one: https://github.com/mui/material-ui/issues/44237
